### PR TITLE
Fix some errors with --disallow-any-generics

### PR DESCRIPTION
**Pull request by gvanrossum**
Trying to help with #3267. I'm tackling all of stdlib/2and3 (and done with that now).

### CODE REVIEW

File: stdlib/2and3/operator.pyi [hunk 1]
```diff
@@ -34,8 +34,8 @@ def is_(a: Any, b: Any) -> bool: ...
 
 def is_not(a: Any, b: Any) -> bool: ...
 
-def abs(x: SupportsAbs) -> Any: ...
-def __abs__(a: SupportsAbs) -> Any: ...
+def abs(x: SupportsAbs[Any]) -> Any: ...
+def __abs__(a: SupportsAbs[Any]) -> Any: ...
```
Comment (ilevkivskyi) [inline stdlib/2and3/operator.pyi]: IIUC these can be turned into `SupportsAbs[_T] -> _T`

Comment (gvanrossum) [reply stdlib/2and3/operator.pyi]: Good point, done.

**Comment by gvanrossum** [comment]
Hm, not sure what's up with pytype... Presumably it's on `Dict[Union[int, str], Union[str, int]]`.

**Comment by srittau** [comment]
pytype is complaining about `Union` in `shutil`, so maybe try to move `None` out of the `IgnoreFn` union? Also cc @rchen152

**Pull request merged**